src/seeders/20200222234112-requests.js

```diff
@@ -73,6 +73,20 @@ module.exports = {
         managerId: '79660e6f-4b7d-4d21-81ad-74f64e9e1c8a',
         createdAt: new Date(),
         updatedAt: new Date(),
+      },
+      {
+        id: 't1e74db7-h610-4f50-9f45-e2371j331ld7',
+        location: 'Koston',
+        destination: 'Kigali',
```

Comment:
Since, on other seeders, we chose to go with lowercase, I believe it should be the case for lines 79 and 80 too.

Reply:
I changed to lowercase